sandbox-template.md

```diff
@@ -0,0 +1,162 @@
+# CNCF SIG Sandbox Project Request Template & Promotion Template
+
+
+# <Project> 
+
+
+## <Promotion/Acceptance> to CNCF <Incubation/Sandbox>
+
+**Primary aim: This template helps a project to prepare for a presentation to the SIG, and then helps the SIG share the collated information with the TOC.**
+
+**Authors:**
+
+
+# Background
+
+**_Link to TOC PR_**
+
+**_Link to Presentation_**
+
+**_Link to GitHub project_**
+
+
+## Project Goal
+
+
+## Current Status
+
+
+## Future Plans
+
+
+# Project Scope
+
+
+## Clear project definition
+
+**_Does the project have a clear and well defined scope_**
+
+## Value-add to the CNCF ecosystem
+
+**_Does the project have a clear value add to the current project ecosystem. How does it relate to other projects with overlapping capabilities._**
+
+
+## Alignment with other CNCF projects
+
+**_Does the project align and actively collaborate with other CNCF projects?_**
+
+**_Does the project require any specific versions of projects (or APIs) to interoperate?  (e.g. K8s API, CSI, CNI, CRI)?_**
+
+**_Does the project augment or benefit other CNCF projects?_**
+
+## Anticipated use cases
+
+**_What problems does the project solve?_**
+
+## Alignment with SIG Reference Model
+
+**_Does the project align with the SIG CNCF reference model and which capabilities does it require/provide at each level of the reference model._**
+
+
+## High level architecture
+
+**_Describe the overall architecture of the project. Feel free to add diagrams._**
+
+# Formal Requirements
+
+**_Document that the project fulfills the requirements as documented in the [CNCF graduation criteria for sandbox](https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc#sandbox-stage)_**
+
+**_Are there any anticipated issues with any of the criteria ?_**
+
+
+# CNCF IP Policy
```

Comment:
I would think this section should be in the proposal, where the project would assert the current status and that when additional compliance steps are planned, if accepted